pyproject.toml

```diff
@@ -195,6 +195,7 @@ exclude = [
     "tests/examples_methods_syntax",
     "tests/test_transformed_data.py",
 ]
+extend-include = ["*.ipynb"]
```

Comment:
Will probably raise this in another PR, not intending to keep this here once out of draft

Reply:
No longer needed since #3539, now a default setting for `ruff`